pkg/autoscaler/autoscaler.go

```diff
@@ -262,21 +262,7 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 		existingMachineSets, err := framework.GetMachineSets(client)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(len(existingMachineSets)).To(BeNumerically(">=", 1))
-
-		By("Getting existing machines")
-		existingMachines, err := framework.GetMachines(client)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(len(existingMachines)).To(BeNumerically(">=", 1))
-
-		By("Getting existing nodes")
-		existingNodes, err := framework.GetNodes(client)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(len(existingNodes)).To(BeNumerically(">=", 1))
-
 		glog.Infof("Have %v existing machinesets", len(existingMachineSets))
-		glog.Infof("Have %v existing machines", len(existingMachines))
-		glog.Infof("Have %v existing nodes", len(existingNodes))
-		Expect(len(existingNodes) == len(existingMachines)).To(BeTrue())
```

Comment:
this appears to be another sanity check, additionally if we start to make the autoscaler tests parallel there could be flakes around the timing of queries for nodes and machines.

Reply:
Having tests that are easier to reason about, especially if things go wrong, is a bigger priority for me than running tests in parallel.  Will removing these log lines make ascertaining what happened if the tests fail more difficult in the future in any way?

Reply:
i don't think removing these will make it more difficult. this test specifically creates 3 machinesets and then reports on the sizes of those machinesets as it operates. i don't think knowing the initial number of machines and nodes adds much, especially when we start to consider that other tests could affect these numbers which would make them even more difficult to interpret, imo.